aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -371,7 +371,7 @@ void upsample_nearest1d_backward_kernel_impl(
     Tensor& grad_input,
     const Tensor& grad_output,
     c10::optional<double> scales_w) {
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(grad_output.scalar_type(), "upsample_nearest1d_backward", [&] {
+  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::Byte, grad_output.scalar_type(), "upsample_nearest1d_backward", [&] {
```

Comment:
I'm not sure if we need backward kernels for `uint8` dtype. @gchanan what did we do for convolutions with `int64` type?

Reply:
Yes, you are right. Now that you talk about it. It does not make sense to have them. Gradients are not going to be `uint8`. 

Changing that